### PR TITLE
6.2.1

### PR DESCRIPTION
**Pull request by minrk**
6.2.0 enables/installs extensions at install time via data_files, so there is no longer a need to duplicate extension code here.

Checklist

* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,32 +11,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: {{ build }}
+  number: 0
   entry_points:
     - ipcluster = ipyparallel.apps.ipclusterapp:launch_new_instance
     - ipcontroller = ipyparallel.apps.ipcontrollerapp:launch_new_instance
     - ipengine = ipyparallel.apps.ipengineapp:launch_new_instance
+  script: pip install --no-deps .
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should we add `--ignore-installed` as well?

ref: https://conda-forge.org/docs/meta.html#use-pip

**Comment by jakirkham** [comment]
Anything else we need to do with the Notebook extensions?

**Comment by nehaljwani** [comment]
Oh no!
```
(test) [nwani@somemachine recipe]$ curl -sLO https://files.pythonhosted.org/packages/7a/7d/9ce244dd14b0299558900ff887cb6c0310d4f48222afb75ecc2fc34b47b4/ipyparallel-6.2.0.tar.gz
 
(test) [nwani@somemachine recipe]$ tar -xf ipyparallel-6.2.0.tar.gz 

(test) [nwani@somemachine recipe]$ cat ipyparallel-6.2.0/ipyparallel.egg-info/SOURCES.txt  | grep benjaminrk
/Users/benjaminrk/dev/ip/parallel/etc/ipyparallel-nbextension.json
/Users/benjaminrk/dev/ip/parallel/etc/ipyparallel-serverextension.json
/Users/benjaminrk/dev/ip/parallel/ipyparallel/nbextension/static/clusterlist.css
/Users/benjaminrk/dev/ip/parallel/ipyparallel/nbextension/static/clusterlist.js
/Users/benjaminrk/dev/ip/parallel/ipyparallel/nbextension/static/main.js
```

File: recipe/meta.yaml (hunk 1 above)
Comment (minrk) [reply recipe/meta.yaml]: Can do. It doesn't do anything extra since the package isn't installed, but added for consistency with the docs.

**Comment by jakirkham** [comment]
What happens if we just delete `ipyparallel.egg-info` first?

**Comment by minrk** [comment]
I just made 6.2.1 that should fix the absolute paths that only come up on Windows. Always a new frustration with setuptools...

**Comment by jakirkham** [comment]
Thanks @minrk. Sorry for the frustration.

**Comment by minrk** [comment]
> Anything else we need to do with the Notebook extensions?

No, the default install should take care of it. There's a test here that should verify that it worked right; I didn't remove that, so it should still be enabling the extension.

**Pull request merged**